### ISSUE TITLE
Warning on unrecognized check name

### ISSUE THREAD

**Issue by TomasMikula** [HairyFotr/linter#47]
```scala
val x = 1/0 // linter:ignore DivideByZeor
```

should issue a warning about unrecognized check (notice the typo).
